src/common/components/Input/Input.tsx

```diff
@@ -14,17 +16,22 @@ export const Input = ({
   leftContent,
   rightContent,
   wrapperProps,
+  label,
+  helperText,
+  isError = false,
   ...props
 }: InputProps) => {
   const { input: inputClasses, wrapper } = inputTheme({ className, size });
   return (
-    <div
-      {...wrapperProps}
-      className={wrapper({ className: wrapperProps?.className })}
-    >
-      {leftContent}
-      <input {...props} className={inputClasses({ className })} />
-      {rightContent}
-    </div>
+    <Field label={label} isError={isError} helperText={helperText}>
```

Comment:
Can you also expose Field props so we can override the class and other props?

Reply:
sure! added at [`96f7043`](https://github.com/AfterClass-io/afterclass.io-v2/pull/27/commits/96f70435615135fa77fe2152278bd09a7bd39112)